src/site/apt/development_json_document.apt

```diff
@@ -0,0 +1,162 @@
+ 
+*** <<<0, 4, 98, 111, 111, 107, 109, 0, 0, 0, 3, 0, 0, 0, 19, 0, 0, 0, 46, 0, 0, 0, 98, 0, 4, 105, 115, 98, 110, 14, 0, 10, 48, 56, 49, 50, 52, 49, 54, 49, 51, 57, 0, 5, 116, 105, 116, 108, 101, 4, 0, 17, 88, 81, 117, 101, 114, 121, 32, 75, 105, 99, 107, 32, 83, 116, 97, 114, 116, 0, 7, 97, 117, 116, 104, 111, 114, 115, 108, 0, 0, 0, 2, 0, 0, 0, 17, 0, 0, 0, 30, 4, 0, 14, 74, 97, 109, 101, 115, 32, 77, 99, 71, 111, 118, 101, 114, 110, 4, 0, 10, 74, 97, 109, 101, 115, 32, 76, 105, 110, 110>>>
+
+  The size of the string is <<4>>. The string value is <<book>>. The value of the key is an Object.
+ 
+  Object in OP(value):
+  
+  In this OP, the value  is again an Object Pointable 
+  (OP) ({{OP:0}}). <<109>> is the <Value Tag> for Object Pointable. 
+   
+  Following are the byte contents for the Object Pointable 
+  ({{OP:0}}). The byte array break down is explained further ahead.
+ 
+*** <<<109, 0, 0, 0, 3, 0, 0, 0, 19, 0, 0, 0, 46, 0, 0, 0, 98, 0, 4, 105, 115, 98, 110, 14, 0, 10, 48, 56, 49, 50, 52, 49, 54, 49, 51, 57, 0, 5, 116, 105, 116, 108, 101, 4, 0, 17, 88, 81, 117, 101, 114, 121, 32, 75, 105, 99, 107, 32, 83, 116, 97, 114, 116, 0, 7, 97, 117, 116, 104, 111, 114, 115, 108, 0, 0, 0, 2, 0, 0, 0, 17, 0, 0, 0, 30, 4, 0, 14, 74, 97, 109, 101, 115, 32, 77, 99, 71, 111, 118, 101, 114, 110, 4, 0, 10, 74, 97, 109, 101, 115, 32, 76, 105, 110, 110>>>
+
+=========================================================================
+** Object ({OP:0})
+  
+  This child is contained in the parent {{Object}}.
+   
+  Byte Array for Object (OP:0)
+  
+***  <<<109, 0, 0, 0, 3, 0, 0, 0, 19, 0, 0, 0, 46, 0, 0, 0, 98, 0, 4, 105, 115, 98, 110, 14, 0, 10, 48, 56, 49, 50, 52, 49, 54, 49, 51, 57, 0, 5, 116, 105, 116, 108, 101, 4, 0, 17, 88, 81, 117, 101, 114, 121, 32, 75, 105, 99, 107, 32, 83, 116, 97, 114, 116, 0, 7, 97, 117, 116, 104, 111, 114, 115, 108, 0, 0, 0, 2, 0, 0, 0, 17, 0, 0, 0, 30, 4, 0, 14, 74, 97, 109, 101, 115, 32, 77, 99, 71, 111, 118, 101, 114, 110, 4, 0, 10, 74, 97, 109, 101, 115, 32, 76, 105, 110, 110>>>
+  
+  <<<109>>> is the value tag for the Object Pointable.
+ 
+  Following are the bytes and contents of the Object Pointable.
+  
+  <<< 0, 0, 0, 3 >>>  These 4 bytes represent the
+  number of key-value pairs in the object which is <<3>>.
+  
+  Following are the bytes that represent the offsets of the three key-value pairs.
+  
+  <<< 0, 0, 0, 19 >>> The offset for the key <<isbn>>.
+  
+  <<< 0, 0, 0, 46 >>> The offset for the key <<title>>.
+  
+  <<< 0, 0, 0, 98 >>> The offset for the key <<authors>>.
+ 
+  As above, the following bytes represent the three key-value pairs inside the Object. 
+  
+  <<<0, 4, 105, 115, 98, 110, 14, 0, 10, 48, 56, 49, 50, 52, 49, 54, 49, 51, 57>>>
+  
+  The size of the string is <<4>>. The string value is <<isbn>>. The String Pointable is followed by an Untyped Atomic Type Pointable.
+  The Value Tag of it is <<14>>. Its size is <<10>>. Its value is <<0812416139>>.
```

Comment:
Actually, I used the xml example for that. So, in this example the value tag is 14 because I believe it is an untyped pointable and not a string pointable.